.github/workflows/nose.yml

```
@@ -0,0 +1,26 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest nose
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

```suggestion
        python -m pip install --upgrade pip
        setup.py install
```
Should be enough. Dependecies are already declared in the setup.py files